tests/forte/processors/predictor_test.py

```diff
@@ -0,0 +1,104 @@
+#  Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import unittest
+from ft.onto.base_ontology import Sentence, Token, EntityMention
+from forte.pipeline import Pipeline
+from forte.train_preprocessor import TrainPreprocessor
+from forte.data.readers.conll03_reader import CoNLL03Reader
+from forte.data.data_pack import DataPack
+from forte.data.converter import Converter
+from forte.data.extractor.attribute_extractor import AttributeExtractor
+from forte.data.extractor.seqtagging_extractor import BioSeqTaggingExtractor
+from forte.data.batchers import FixedSizeDataPackBatcherWithExtractor
+from forte.processors.base.batch_processor import Predictor
+
+
+class ProcessingBatcher(unittest.TestCase):
+
+    def setUp(self):
+        # Define and config the Pipeline
+        self.dataset_path = "data_samples/conll03"
+
+    def test_AttributeExtractor(self):
+        pipeline = Pipeline[DataPack]()
+        pipeline.set_reader(CoNLL03Reader())
+        pipeline.initialize()
```

Comment:
Can you create multiple tests that mixup predictor and normal processor like this:
For pipeline:
https://github.com/asyml/forte/blob/master/tests/forte/pipeline_test.py#L231
For multi-pack pipeline:
https://github.com/asyml/forte/blob/master/tests/forte/pipeline_test.py#L453

The goal is to make sure Predictor can intertwine other processors.

A simple way to create the test is to copy those test functions and but replace `DummmyFixedSizeBatchProcessor` with `DummyPredictor`.

This test can be in that file too, so you don't have to design new setup, and data.

Reply:
I will need to think about how to do this. Because Predictor is different from DummmyFixedSizeBatchProcessor. It needs extractor and the extractor needs to be compatible with the dataset file.